Automattic/hostmgr
# Update Commands to use libhostmgr

Builds on #46 by migrating the `hostmgr` CLI commands to use `libhostmgr` under the hood. 

This uses far less code to do the job, and finishes threading `async` support all the way up to the UI layer.

**To Test:**
Running each command should be a good way to test – you can try:

- `swift run hostmgr vm list` [should emit a list of VMs, local and remote]
- `swift run hostmgr vm exists xcode-99` [should exit with an error saying the VM doesn't exist]
- `swift run hostmgr vm clean` [should run with empty output]
- `swift run hostmgr vm fetch xcode-14.2` [should download the VM and unpack it]
- `swift run hostmgr vm start xcode-14.2` [should fire up the VM]
- `swift run hostmgr vm stop --all --immediately` [should immediately pull the plug on the VM]
